### ISSUE TITLE
Getting nan after iteration 1

### ISSUE THREAD

**Issue by htzheng** [dgilton/deep_equilibrium_inverse#1]
Hi, thanks for the code and the paper! I tried to run the script _deblur_proxgrad_fixedeta_pre.py_  but I found that the loss is nan after every iteration. I wonder if there is something that I missed there? Thanks!

**Comment by zaccharieramzi** [comment]
@dgilton also getting the same error here: maybe the settings of the current file are not the actual ones.

**Comment by zaccharieramzi** [comment]
Apparently taking the setting mentionned in the paper with 50 Anderson steps (instead of 100 by default in the code) works without giving nans, so I am going to stick to that

**Comment by TOT-code** [comment]
@zaccharieramzi Hello, I also encountered this problem. Is pre-training of the denoiser necessary? Can you share your parameter settings at that time? Thank you so much.

**Comment by zaccharieramzi** [comment]
I don't remember exactly but I don't think so: I think it's fine to just do it from scratch but you need to go for 50 anderson steps rather than 100
However, I remember talking with @dgilton and IIUC there is not going to be a lot of maintenance of this repo
